_includes/components/card-vertical.html

```diff
@@ -10,6 +10,16 @@
 - type {Array}
 {% endcomment %}
 
+{% if include.iconIndex == 0 %}
+  {% assign iconName = "project-bar" %}
+{% elsif include.iconIndex == 1 %}
+  {% assign iconName = "project-line" %}
+{% elsif include.iconIndex == 2 %}
+  {% assign iconName = "project-database" %}
+{% else %}
+  {% assign iconName = "project-bar" %}
+{% endif %}
```

Comment:
Updates from design to help vary the imagery used for projects in addition to the background color.